modules/priority-class/main.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  priority_class_default = [
+    {
+      name  = "high"
+      value = "1000000"
+    },
+    {
+      name  = "medium"
+      value = "500000"
+    },
+    {
+      name  = "low"
+      value = "250000"
+    }
+  ]
+  merged_priority_class = concat(local.priority_class_default, var.priority_class)
+  priority_class        = [for map in local.merged_priority_class : map if length(map) > 0]
```

Comment:
if we set default value and type (instead of "any") there should not be need this filtration